IDR/production/2.visualize_stats.py

```diff
@@ -0,0 +1,61 @@
+import pandas as pd
+from plotnine import *
```

Comment:
oh nice! I hadn't realized you were using plotnine (I thought those were pure ggplot2 figures)